cmd/neofs-node/cache.go

```diff
@@ -312,41 +312,41 @@ func (s *ttlContainerLister) List(id *user.ID) ([]cid.ID, error) {
 // All described race cases aren't critical since cache values expire anyway, we just try
 // to increase cache actuality w/o huge overhead on synchronization.
 func (s *ttlContainerLister) update(owner user.ID, cnr cid.ID, add bool) {
-	strOwner := owner.EncodeToString()
+	for _, key := range []string{"", owner.EncodeToString()} {
```

Comment:
`""` for "owner-less" containers? Why in this order then? I'd expect a check for direct `owner` hit first and a fallback then.

Reply:
Actually, we need both. `continue` inside the cycle is just for the case when we do not cache the result.

Reply:
Can `""` case grow a lot in some cases? What if a container is cached is owner-less, but then is added with an owner? Something's fishy here.

Reply:
Nothing bad happens -- both lists are completely independent.
I have removed caching for the full-list because on close inspection we do not really use it that often.